frontend/app/module/DevHomePage.tsx

```diff
@@ -44,6 +44,9 @@ export function DevHomePage() {
               <li>
                 <Link to={`/elections/1#coordinator`}>Verkiezing 1</Link>
               </li>
+              <li>
+                <Link to={`/elections/1/status#coordinator`}>Statusoverzicht steminvoer Verkiezing 1</Link>
```

Comment:
```suggestion
                <Link to={`/elections/1/status#coordinator`}>{t("election_status.main_title")} {t("election.election")} 1</Link>
```
Needs to be updated if https://github.com/kiesraad/abacus/pull/699 is merged first

Reply:
#699 hasn't been merged yet, merging this PR first.